setup.cfg

```diff
@@ -23,9 +23,9 @@ classifiers =
 [options]
 packages = find:
 install_requires =
-    aiida-core~=1.0
+    aiida-core>=1.0,<3
     aiidalab>=21.11.2
-    aiidalab-eln~=0.1
+    aiidalab-eln~=0.1.2
```

Comment:
Any reason for increasing stringency here?

Reply:
no, I just wanted to increase the minimum version; let me fix this